src/ipc-dbus/DBusIPCAPI_v1.cpp

```diff
@@ -1028,19 +1028,28 @@ DBusIPCAPI_v1::interface_config_gateway_handler(
 ) {
 	DBusHandlerResult ret = DBUS_HANDLER_RESULT_NOT_YET_HANDLED;
 	dbus_bool_t default_route = FALSE;
+	dbus_bool_t preferred = TRUE;
+	dbus_bool_t slaac = TRUE;
+	dbus_bool_t on_mesh = TRUE;
 	uint32_t preferred_lifetime = 0;
 	uint32_t valid_lifetime = 0;
 	uint8_t *prefix(NULL);
 	int prefix_len_in_bytes(0);
 	struct in6_addr address = {};
 	bool did_succeed(false);
+	int16_t priority_raw;
+	NCPControlInterface::OnMeshPrefixPriority priority(NCPControlInterface::PREFIX_MEDIUM_PREFERENCE);
 
 	did_succeed = dbus_message_get_args(
 		message, NULL,
 		DBUS_TYPE_BOOLEAN, &default_route,
 		DBUS_TYPE_ARRAY, DBUS_TYPE_BYTE, &prefix, &prefix_len_in_bytes,
 		DBUS_TYPE_UINT32, &preferred_lifetime,
 		DBUS_TYPE_UINT32, &valid_lifetime,
+		DBUS_TYPE_BOOLEAN, &preferred,
```

Comment:
This changes the dbus message format and  breaks compatibility with sw using existing version (using existing dbus message format).

We should check for the case where the new parameters are not provided, which should cause a `did_succeed` failure, and then parse it up to old parameters (use default values for new ones).  `DBusIPCAPI_v1::interface_route_add_handler()` is an example of how it's done for another command.